tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_ca.yml

```diff
@@ -14,5 +14,12 @@
 - name: Importing client certificate for OCSP
   shell: certutil -L -d /var/lib/pki/{{ topology }}-CA/alias -n "{{ nickname_ocsp.stdout }}" -a > /tmp/test_dir/ocsp_signing.crt
 
-- name : Starting CA Subsystem
+- name: Set debug log level to 5
+  lineinfile:
```

Comment:
Minor: You can use the following CLI tool to set the debug level instead of modifying the CS.cfg manually:
````
pki-server ca-config-set debug.level 5
````
Note that you still need to restart the server after you run the above command.

Also, IMO, when "debug" flag is set, it is more useful to collect ALL logs (including TRACE) logs (ie) the debug level should be set to `1`

Ref: https://www.dogtagpki.org/wiki/PKI_10.7_Server_Debug_Log

Reply:
Currently i have excluded the playbook changes for another MR. So marking this comment resolved for this MR.
This MR consists changes related to executing existing tests in Fedora32.